test/flutter_redux_test.dart

```diff
@@ -164,6 +164,32 @@ void main() {
       expect(numBuilds, 2);
     });
 
+    testWidgets(
+        'builds the latest state of the store after a change event is triggered from initState',
```

Comment:
Let's not test this (rebuilds are covered by other test). Instead, let's make a test called `"can access store from initState`. The test should create a `StatefulStoreCaptor` nested inside a `StoreProvider`.

The `StatefulStoreCaptor` should not be given any state, but it should rather capture the store in the state object during `initState`.

It should then check whether the store it captures is the same as the store created at the beginning of this test. You can use a `GlobalKey` to access state variables.

The final check in this test should be `expect(expect(StatefulStoreCaptor.captorKey.currentState.store, store);`